Implement CLI / Config with Cobra / Viper (#1043)
ansible/receptor: docs/source/getting_started_guide/index.rst

Getting started with Receptor
#############################

> Receptor is an overlay network intended to ease the distribution of work across a large and dispersed collection of workers. Receptor nodes establish peer-to-peer connections with each other via existing networks. Once connected, the receptor mesh provides datagram (UDP-like) and stream (TCP-like) capabilities to applications, as well as robust unit-of-work handling with resiliency against transient network failures.

Reviewer: ```suggestion
Receptor is an overlay network that distributes work across large and dispersed collections of worker nodes.
Receptor nodes establish peer-to-peer connections through existing networks.
Once connected, the Receptor mesh provides datagram (UDP-like) and stream (TCP-like) capabilities to applications, as well as robust unit-of-work handling with resiliency against transient network failures.
```

Reviewer: Just a nit really but I prefer one sentence per line. RST will form paragraphs from concurrent sentences automatically. One sentence per line has a few benefits, one of the main ones is ease of review.

Reviewer: I'd also apply the suggestions to the long description in the `root.go` file too https://github.com/ansible/receptor/pull/1043/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR23
